configs/milvus.yaml

```diff
@@ -126,6 +126,12 @@ pulsar:
 #   saslMechanisms: PLAIN
 #   securityProtocol: SASL_SSL
 #   readTimeout: 10 # read message timeout in seconds
+#   ssl:
```

Comment:
Using TLS as an option name would be better, SSL options already exist under kafka?